pages/reference/supervisor/bandwidth-reduction.md

```diff
@@ -4,52 +4,41 @@ title: Reduce bandwidth usage
+**Cloudlink VPN** Defines the ability to send instantaneous updates to the device. Turning off cloudlink means that any configuration or variable update is reflected only when the device polls for these changes. The Web Terminal does not function when cloudlink is disabled. This also disables the public URL functionality.
 
-Data usage impact
------------------
+**Device Metrics** [Device metrics][device-metrics] such as CPU temperature and memory usage won't be reported to the API, and won't be displayed on the summary page of the device dashboard.
 
-| Service                                                                 | Usage (Including DNS overhead)                          | Related Configuration Variable       |
-|-------------------------------------------------------------------------|---------------------------------------------------------|--------------------------------------|
-| API poll (Once per 15min)                                               | 6650 Bytes per request                                  | BALENA_SUPERVISOR_POLL_INTERVAL      |
-| {{ $names.company.lower }} Supervisor update poll (Once every 24 hours) | 6693 Bytes per request                                  | Not configurable via variables       |
-| cloudlink enabled                                                       | 43 Bytes / second                                       | BALENA_SUPERVISOR_VPN_CONTROL        |
-| TCP check cost (When cloudlink disabled)                                | 47.36 Bytes / second                                    | BALENA_SUPERVISOR_CONNECTIVITY_CHECK |
-| Hardware metrics reporting                                              | ~66 Bytes every 10 seconds when there have been changes | BALENA_SUPERVISOR_HARDWARE_METRICS   |
+**API Poll Interval** Defines the time interval between device checks for updates to the fleet, for example new code pushes, environment variables changes, or cloudlink control changes for the device. If cloudlink is disabled, lengthening this interval increases the latency of propagating these updates to the device.
 
-Example minimum bandwidth settings
-----------------------------------
+**NetworkManager connectivity check** ??? What happens if it fails?
 
-The following settings lead to data usage of approximately 1.3MB per month:
+**Device Logging** Any logs written by the user container or the device Agent are not sent to the dashboard when this variable is set to false.
+
+**Cloudlink connectivity check** Defines the device's ability to test and indicate (via an LED when available) that it has issues with connectivity.
 
-* Disable `BALENA_SUPERVISOR_VPN_CONTROL`
-* Disable `BALENA_SUPERVISOR_CONNECTIVITY_CHECK`
-* Change `BALENA_SUPERVISOR_POLL_INTERVAL` to 24 hours (86400000 ms)
-* Disable `BALENA_SUPERVISOR_LOG_CONTROL`
-* Disable `BALENA_SUPERVISOR_HARDWARE_METRICS`
 
```

Comment:
Could we also add a section to recommend users to use our Metrics block to check bandwidth usage if they want. Could be a good pitch to use it on their fleet or at least some devices?

Shows that we trust our results and look forward to more data to improve this? What say

Reply:
I'm a little confused here. I did suggest above that users verify for themselves. The linked project uses the Metrics block from Hub.

> You can perform these measurements yourself as well!

Reply:
That is true @kb2ma Yeah, let's leave this one out. Since there is a getting started section on the network tool